build/uidp_helm/templates/ingress.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: {{ .Values.ingress.name }}
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+spec:
+  rules:
+  - host: {{ .Values.ingress.host }}
+    http:
+      paths:
+      - path: /newuser
+        backend:
+          serviceName: {{ .Values.service.uidpName }}
+          servicePort: {{ .Values.service.portOuter }}
+      - path: /deleteuser
+        backend:
+          serviceName: {{ .Values.service.uidpName }}
+          servicePort: {{ .Values.service.portOuter }}
+      - path: /updatepwd
+        backend:
+          serviceName: {{ .Values.service.uidpName }}
+          servicePort: {{ .Values.service.portOuter }}
+      - path: /login
+        backend:
+          serviceName: {{ .Values.service.uidpName }}
+          servicePort: {{ .Values.service.portOuter }}
+      - path: /refreshtoken
+        backend:
+          serviceName: {{ .Values.service.uidpName }}
+          servicePort: {{ .Values.service.portOuter }}
```

Comment:
Change the variable name to be just port, and the port of the pod to be targetPort